modules/tinymce/src/models/dom/main/ts/table/api/TableCellSelection.ts

```diff
@@ -187,7 +185,15 @@ export const TableCellSelection = (editor: Editor): TableCellSelection => {
   const clear = (container: Node) =>
     annotations.clear(SugarElement.fromDom(container));
 
-  return {
-    clear
-  };
+  editor.addCommand('TableCellSelectionClear', (_ui, container: Node) => {
```

Comment:
I wonder if this should instead be a model api function and also a selection api function. Since you have a function to get the selected cells. You could have a editor.selection.clearSelectedCells(). In the alt model it could be just a noop at this point.